fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -269,6 +271,19 @@ ErrorOr<Optional<TenantMapEntry>> getTenantEntry(ProxyCommitData* commitData,
 	return Optional<TenantMapEntry>();
 }
 
+ErrorOr<Optional<TenantName>> getTenantName(ProxyCommitData* commitData, int64_t tenantId, bool logOnFailure) {
```

Comment:
It looks like you are only ever calling this with logOnFailure=true. Do we need this as a parameter?